### DISCUSSION TITLE
New unstoppable notification email about new org team members

### DISCUSSION THREAD

**Discussion by zpao** [community/community#21919] [New to GitHub]
<p>Starting Monday, whenever a person is added to a team in one of the organizations that I manage, each owner in the organization receives an email about that addition. There doesn’t appear to be a setting to opt-out of this email either at the personal level nor the organization level. My colleagues and I manage orgs totallying over 500 teams so this is going to get noisy fast.</p>
<p>This notification also doesn’t follow the same pattern most emails follow with useful headers and CCs (<a href="https://help.github.com/articles/about-email-notifications/" rel="nofollow noopener">https://help.github.com/articles/about-email-notifications/</a>), so it’s not particularly easy to filter these straight to the trash.</p>
<p> Is there a setting that we missed? Can we get an opt-out built in sometime soon?</p>

**Comment by Wabri** [comment]
<p>Is the email not from <span>noti</span><span>fications@</span><span><a href="http://github.com" rel="nofollow noopener">github.com</a>?<br></span></p>

**Comment by zpao** [comment, marked as answer]
<p>Support responded to me - GitHub disabled this feature after community feedback.</p>
